docs/Rakefile

```diff
@@ -19,17 +19,19 @@ task test: :build do
     "./_site",
     parallel: { in_threads: 4 },
     favicon: true,
-    http_status_ignore: [0, 403],
+    ignore_status_codes: [0, 403],
     assume_extension: true,
     check_favicon: true,
     check_opengraph: true,
     check_html: true,
     check_img_http: true,
     enforce_https: true,
-    file_ignore: [
+    check_external_hash: false,
+    check_internal_hash: false,
```

Comment:
Might be worth keeping these in and whitelisting e.g. GitHub.com usage. My settings, if they are useful: https://github.com/MikeMcQuaid/mikemcquaid.com/blob/master/Rakefile